### PR TITLE
Updates README.md with cron-event-source and added "Noisy Neighbor" the the list

### LINKED ISSUE #1851: README.md should mention cron event source
The README.md currently lists other official brigade projects that are compatible with Brigade v2.

idk if we'll eventually get away from doing that, but for now, the cron event source is conspicuously absent from the page.

Let's add it: https://github.com/brigadecore/brigade-cron-event-source

### PR DESCRIPTION
**Pull request by hamees-sayed**
Fixes #1851

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**If applicable**:
- [x] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,25 +41,25 @@ and convert them to Brigade events that are emitted into Brigade's event bus.
 * [GitHub Gateway](https://github.com/brigadecore/brigade-github-gateway)
 * [Slack Gateway](https://github.com/brigadecore/brigade-slack-gateway)
 
+### Other Event Sources
+
+* [Cron-Event-Source](https://github.com/brigadecore/brigade-cron-event-source)
+* [Brigade Noisy Neighbor](https://github.com/brigadecore/brigade-noisy-neighbor)
+  (The Brigade team is utilizing "Brigade Noise Neighbor" to keep our own internal 
+  Brigade installation under a steady load. We hope the larger event volumes than 
+  what we generate on our own will help us to identify and resolve bugs sooner.)
```
Comment (krancour) [inline README.md:50]: @hamees-sayed let's drop this text while we're at it.

File: README.md [hunk 2]
```diff
@@ -41,25 +41,25 @@ and convert them to Brigade events that are emitted into Brigade's event bus.
 * [GitHub Gateway](https://github.com/brigadecore/brigade-github-gateway)
 * [Slack Gateway](https://github.com/brigadecore/brigade-slack-gateway)
 
+### Other Event Sources
+
+* [Cron-Event-Source](https://github.com/brigadecore/brigade-cron-event-source)
```
Comment (krancour) [inline README.md:46]: Can we remove the dashes in the name please?

**Comment by krancour** [comment]
@hamees-sayed thanks for the contribution!

Is there a chance you could please add a [DCO signoff](https://github.com/brigadecore/community/blob/main/contributing.md#dco-sign-off) to your commits? lmk if you need help with that.

File: README.md (hunk 1 above)
Comment (hamees-sayed) [reply README.md:50]: Yup definitely, I remove the dash and this text

File: README.md [hunk 3]
```diff
@@ -41,25 +41,22 @@ and convert them to Brigade events that are emitted into Brigade's event bus.
...
-larger event volumes than what we generate on our own will help us to identify
-and resolve bugs sooner.
-
 ### SDKs
 
 Use any of these to develop your own integrations!
 
 * [Brigade SDK for Go](https://github.com/brigadecore/brigade/tree/main/sdk) (used by Brigade itself)
-* [Brigade SDK for JavaScript](https://github.com/krancour/brigade-sdk-for-js) (and TypeScript)]
+* [Brigade SDK for JavaScript](https://github.com/krancour/brigade-sdk-for-js) (and TypeScript)
```
Comment (krancour) [inline README.md:59]: Nice catch. Thanks for fixing this typo also.

**Pull request merged**